README.md

```diff
@@ -59,3 +60,14 @@
 - [neo-gui](https://github.com/neo-project/neo-gui/) - Official wallet built by NEO team, which has a UI.
 - [neon-wallet](https://github.com/CityOfZion/neon-wallet) - Light wallet for the NEO blockchain.
 - [neotracker.io](https://neotracker.io/wallet) - NEO web wallet.
+
+## Information blogs
+
+- [The NEO Subreddit](https://www.reddit.com/r/NEO/).
```

Comment:
Remove `The` and add a description after the link as some users may not know about Reddit.